google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -873,18 +873,17 @@ public void testListPartitions() throws InterruptedException {
     Job job = bigquery.create(JobInfo.of(QueryJobConfiguration.newBuilder(query).build()));
     job.waitFor();
     assertTrue(job.isDone());
-    try {
-      Map<String, Object> row = new HashMap<String, Object>();
-      row.put("StringField", "StringValue");
-      InsertAllRequest request = InsertAllRequest.newBuilder(tableId).addRow(row).build();
-      InsertAllResponse response = bigquery.insertAll(request);
-      assertFalse(response.hasErrors());
-      assertEquals(0, response.getInsertErrors().size());
-      List<String> partitions = bigquery.listPartitions(TableId.of(DATASET, tableName));
-      assertEquals(1, partitions.size());
-    } finally {
-      bigquery.delete(tableId);
-    }
+    Table table = bigquery.getTable(tableId);
+    System.out.println(table.getTableId().getTable());
```

Comment:
`System.out.println(table.getTableId().getTable());`  -> please remove this statement

Reply:
@athakor done